Extension/src/LanguageServer/configurations.ts

```diff
@@ -46,6 +46,7 @@ export interface ConfigurationJson {
 export interface Configuration {
     name: string;
     compilerPath?: string;
+    compilerInfo?: CompilerInfo[];
```

Comment:
I usually use plurals in the variable names for arrays/lists.  e.g. consider renaming to `'compilers` or `knownCompilers` (`compilerInfos` sounds weird)